manager/integration/README.md

```diff
@@ -6,6 +6,8 @@ Requirement:
 1. A Kubernetes cluster with at least 3 nodes
 2. Longhorn system has already been successfully deployed in the cluster.
 3. No volume exists in the Longhorn system.
+4. Need kubernetes 1.10 or higher.
+5. Need to enable `feature-gates: MountPropagation=true` for kube-api and kubelet while using RKE
```

Comment:
Don't need to mention `while using RKE`. The issue has been fixed on RKE. We can just say:

E.g. 
5. Make sure `MountPropagation` feature gate is enabled
5.1. For RKE before v0.1.9, you would need the extra parameter `feature-gates: MountPropagation=true` for `kube-api` and `kubelet` to enable the feature gate.